cargo-clone/src/main.rs

```diff
@@ -77,9 +78,9 @@ fn main() {
         .arg(Arg::with_name("directory").help("The destination directory. If it ends in a slash, crates will be placed into its subdirectories.").last(true));
 
     let matches = app.get_matches();
-    let mut config = Config::default().expect("Unable to get config.");
 
-    if let Err(e) = execute(matches, &mut config) {
+    if let Err(e) = execute(&matches) {
+        let config = cargo_config(&matches).expect("Unable to get config.");
```

Comment:
Here I create a new cargo_config. Before, you were reusing the existing one.
Is it a problem?

Reply:
I think this is fine.